third-parties/scroll/database/migrate/migrations/00004_chunk.sql

```diff
@@ -35,7 +35,7 @@ create table chunk
 );
 
 comment
-on column chunk.proving_status is 'undefined, unassigned, skipped, assigned, proved, verified, failed';
+on column chunk.proving_status is 'undefined, unassigned, assigned, proved, verified, failed';
```

Comment:
Is `proving_status` used anywhere in `rollup-explorer-backend`?

Reply:
No.